stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -539,7 +738,10 @@ impl NakamotoBlockHeader {
     /// - At least the minimum number of signatures (based on total signer weight
     /// and a 70% threshold)
     /// - Order of signatures is maintained vs signer set
-    pub fn verify_signer_signatures(&self, reward_set: &RewardSet) -> Result<(), ChainstateError> {
+    ///
+    /// Returns the signing weight on success.
+    /// Returns ChainstateError::InvalidStacksBlock on error
+    pub fn verify_signer_signatures(&self, reward_set: &RewardSet) -> Result<u32, ChainstateError> {
```

Comment:
Based on the timeout mutant output ([issue here](https://github.com/stacks-network/stacks-core/issues/4960#issuecomment-2223018774)), above the doc comments of the  `verify_signer_signatures` function should be added this line
```
#[cfg_attr(test, mutants::skip)])
``` 
so the function is skipped when performing mutation testing. This one is a bit strange, I don't see any http calls inside it, but it timeouts and I think this only happens because it wouldn't expect a response as low as 0 or 1.

Reply:
Yeah, that is strange.  It does not do any I/O.  It returns the signing weight of the valid signatures, which will invariably be greater than 1 for a well-formed block.